src/main/resources/META-INF/plugin.xml

```diff
@@ -207,6 +207,14 @@
                          level="WEAK WARNING"
                          implementationClass="me.serce.solidity.ide.inspections.VisibilityModifierInspection"/>
 
+        <localInspection language="Solidity" groupName="Solidity"
+                                 displayName="Member is not defined"
+                                 enabledByDefault="true"
+                                 level="ERROR"
```

Comment:
I did a few rounds of testing on various repositories, and it seems that due the current incomplete resolution strategy, the inspection produces a large number of false positives. Too large to be enabled by default. 

For example, the inspection currently fails for types imported through `import as` or  the following code results in an error:
```solidity
// SPDX-License-Identifier: GPL-3.0

pragma solidity >=0.7.0 <0.9.0;

contract B {
    type ShortString is bytes32;
    
    bytes32 private constant V = 0x00000000000000000000000000000000000000000000000000000000000000FF;

    function a() public pure returns (ShortString) {
        return ShortString.wrap(V);
    }
}
```

What do you think of either only scoping the inspection to the cases where the receiver was actually resolved, and it's not unknown? Or, alternatively, making it a `WEAK WARNING`?

Reply:
hmm, it seems my yesterday reply was lost.
regarding this I did the following:
1. Set `WEAK WARNING` as the default level for inspection 
2. Update inspection to now report an error when the receiver was not resolved
3. Fixed the case with resolving a user defined value type